metadata-db-app/project.clj

```diff
@@ -12,6 +12,7 @@
   :dependencies [[cheshire "5.10.0"]
                  [clj-http "3.11.0"]
                  [clj-time "0.15.1"]
+                 [com.amazonaws/aws-java-sdk-core "1.12.487"]
```

Comment:
where is this used?

Reply:
It was required for building with the faraday library that is being used to communicate with DynamoDB